server/routes/finance-id/financeIdController.ts

```diff
@@ -31,59 +30,47 @@ export default class FinanceIdController {
       // DELETE ASSESSMENT
       if (deleteAssessmentConfirmed) {
```

Comment:
wouldn't it be better to do 
if (deleteAssessmentConfirmed && prisonNumber && assessmentId) ... if prisonNumber is null, after `as string` is still going to be null

I might be wrong, just a suggestion

Reply:
Usually, but I think here it needs to be handle in the middleware since prisonerNumber being null can't possibly be valid in this part of the code.